core/adapters/sqldatawarehouse.ts

```diff
@@ -0,0 +1,109 @@
+import { dataform } from "@dataform/protos";
+import { Task, Tasks } from "../tasks";
+import { Adapter } from "./base";
+import { IAdapter } from "./index";
+
+export class SQLDataWarehouseAdapter extends Adapter implements IAdapter {
+  private project: dataform.IProjectConfig;
```

Comment:
looks like you don't use this? in which case, let's remove it & the constructor.